src/components/landing-page/landing-page.html

```diff
@@ -1,4 +1,18 @@
 <div class="landing">
+  <overlay-panel show-panel="$ctrl.orderingPanelVisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
+    <order-service
+        ng-if="$ctrl.selectedServiceClass"
+        base-project-url="{{$ctrl.baseProjectUrl}}"
```

Comment:
Not sure if this is correct.  Previously, `base-project-url `was only in `<catalog-search>`, but not set in `<catalog-search>`'s controller (...at least not in the catalog app?).  I assume this is set in the console.   If so, will it affect` <ordering-panel> `when launched from `<service-view>` and` <project-summary>` in the console app?